src/container/App.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from "react-redux";
```

Comment:
This app container is not necessary. we are already handling it inside TaskList container